go/vt/vtgate/engine/aggregations.go

```diff
@@ -35,7 +35,12 @@ import (
 // It contains the opcode and input column number.
 type AggregateParams struct {
 	Opcode opcode.AggregateOpcode
-	Col    int
+
+	// Input source specification - exactly one of these should be set:
+	// Col: Column index for simple column references (e.g., SUM(column_name))
+	// EExpr: Evaluated expression for literals, parameters
+	Col   int
+	EExpr evalengine.Expr
```

Comment:
I guess if `EExpr` is set, the value in `Col` is ignored? I'm asking because it says "exactly one should be set", but `0` (the default / empty value fro `Col`) is a valid value and I was confused for a moment what this means then.